frontend/packages/aquascope-editor/src/editor-utils/stepper.tsx

```diff
@@ -0,0 +1,383 @@
+import { Line, StateEffect } from "@codemirror/state";
```

Comment:
Shouldn't this commit also delete `permission-steps.tsx` or whatever the previous file was?